### PR TITLE
docs: sd accordion docs [skip chromatic]

### PR DESCRIPTION
**Pull request by smfonseca**
## Definition of Reviewable:

- [ ] Documentation is created/updated
- [ ] Migration Guide is created/updated
- [ ] E2E tests (features, a11y, bug fixes) are created/updated
- [ ] Stories (features, a11y) are created/updated
- [x] relevant tickets are linked

### CODE REVIEW

File: packages/components/src/components/accordion/accordion.stories.ts [hunk 1]
```diff
@@ -34,54 +38,52 @@ export const Default = {
...
+/**
+ * The `summary` slot allows you to use custom html in the summary.
+ */
+
+export const SlottedSummary = {
+  name: 'Slotted Summary',
+  render: () => html`
+    <div class="grid grid-cols-2">
+      <sd-accordion>
+        <div slot="summary"><h4>Custom summary</h4></div>
```
Comment (mariohamann) [inline packages/components/src/components/accordion/accordion.stories.ts:76]:
```suggestion
        <div slot="summary"><h4>Summary as headline</h4></div>
```

File: packages/components/src/components/accordion/accordion.stories.ts [hunk 2]
```diff
@@ -90,87 +92,15 @@ export const SummaryLength = {
...
-  render: (args: any) => {
-    return html`<div class="mouseless">${generateTemplate({ args })}</div>`;
-  },
-
-  play: async ({ canvasElement }: { canvasElement: HTMLUnknownElement }) => {
-    const el = canvasElement.querySelector('.mouseless sd-accordion');
-    await waitUntil(() => el?.shadowRoot?.querySelector('header'));
-    await userEvent.type(el!.shadowRoot!.querySelector('header')!, '{space}', { pointerEventsCheck: 0 });
-  }
+export const SlottedIcons = {
```
Comment (mariohamann) [inline packages/components/src/components/accordion/accordion.stories.ts:95]: Those slots are meant for theming purposes. E. g. https://www.attrax.lu will use/uses our components as well but they shouldn't the official icons. THerefore I think we should remove this here and make this less "explicit" to not invite people to do this. What do you think @coraliefeil?

File: packages/components/src/components/accordion/accordion.stories.ts (hunk 1 above)
Comment (smfonseca) [reply packages/components/src/components/accordion/accordion.stories.ts]: Updated after sync with @coraliefeil and @MartaPintoTeixeira

File: packages/components/src/components/accordion/accordion.stories.ts (hunk 2 above)
Comment (smfonseca) [reply packages/components/src/components/accordion/accordion.stories.ts]: These have been removed.

File: packages/components/src/components/accordion/accordion.stories.ts [hunk 3]
```diff
@@ -2,20 +2,27 @@
...
 
 const { argTypes, parameters } = storybookDefaults('sd-accordion');
 const { overrideArgs } = storybookHelpers('sd-accordion');
 const { generateTemplate } = storybookTemplate('sd-accordion');
 
+/**
+ *
+ * Accordions show a brief summary and expand to display additional content.
+ *
+ *  **Related templates**:
```
Comment (coraliefeil) [inline packages/components/src/components/accordion/accordion.stories.ts:14]: *  **Related components**:

File: packages/components/src/components/accordion/accordion.stories.ts [hunk 4]
```diff
@@ -2,20 +2,27 @@
...
+ * - [Accordion Group](?path=/docs/templates-accordion-group--docs)
```
Comment (coraliefeil) [inline packages/components/src/components/accordion/accordion.stories.ts:15]: * - [sd-accordion-group](?path=/docs/components-sd-accordion-group--docs)

File: packages/components/src/components/accordion/accordion.stories.ts [hunk 5]
```diff
@@ -2,20 +2,27 @@
...
-import { waitUntil } from '@open-wc/testing-helpers';
-import cx from 'classix';
 
 const { argTypes, parameters } = storybookDefaults('sd-accordion');
 const { overrideArgs } = storybookHelpers('sd-accordion');
 const { generateTemplate } = storybookTemplate('sd-accordion');
 
+/**
+ *
+ * Accordions show a brief summary and expand to display additional content.
```
Comment (coraliefeil) [inline packages/components/src/components/accordion/accordion.stories.ts:12]: * Shows a brief summary and expand to display additional content.

File: packages/components/src/components/accordion/accordion.stories.ts [hunk 6]
```diff
@@ -34,143 +41,43 @@ export const Default = {
...
- * Use the expand-icon and collapse-icon slots to change the expand and collapse icons, respectively.
- * To disable the animation, override the rotate property on the summary-icon part as shown below:
+ * Use `summary` to provide the text in the accordion header.
  *
- * ```
- * sd-accordion.custom-icons::part(summary-icon) {
- *   rotate: none;
- * }
- * ```
+ * __Hint:__ The `summary` slot allows you to use custom html in the summary.
```
Comment (coraliefeil) [inline packages/components/src/components/accordion/accordion.stories.ts:64]: * __Hint:__ The `summary` slot allows you to use custom html.

File: packages/components/src/components/accordion/accordion.stories.ts (hunk 4 above)
Comment (smfonseca) [reply packages/components/src/components/accordion/accordion.stories.ts:15]: Not sure what you mean here. Can you please clarify?

File: packages/components/src/components/accordion/accordion.stories.ts (hunk 5 above)
Comment (smfonseca) [reply packages/components/src/components/accordion/accordion.stories.ts:12]: Updated

File: packages/components/src/components/accordion/accordion.stories.ts (hunk 6 above)
Comment (smfonseca) [reply packages/components/src/components/accordion/accordion.stories.ts:60]: Updated

File: packages/components/src/components/accordion/accordion.stories.ts (hunk 3 above)
Comment (smfonseca) [reply packages/components/src/components/accordion/accordion.stories.ts:14]: Clarified on teams call

File: packages/components/src/components/accordion/accordion.stories.ts (hunk 4 above)
Comment (smfonseca) [reply packages/components/src/components/accordion/accordion.stories.ts:15]: Clarified on teams call

**Comment by coraliefeil** [comment]
Fine with me.

**Review by mariohamann — APPROVED** [review]
@smfonseca 🪗 → 🛳️

File: packages/components/src/components/accordion/accordion.stories.ts [hunk 7]
```diff
@@ -2,20 +2,31 @@
...
-import { waitUntil } from '@open-wc/testing-helpers';
-import cx from 'classix';
 
 const { argTypes, parameters } = storybookDefaults('sd-accordion');
 const { overrideArgs } = storybookHelpers('sd-accordion');
 const { generateTemplate } = storybookTemplate('sd-accordion');
 
+/**
+ *
+ * Shows a brief summary and expands to display additional content.
```
Comment (MartaPintoTeixeira) [inline packages/components/src/components/accordion/accordion.stories.ts:12]: Used to show a brief summary and expand to display additional content.

**Pull request merged**